### ISSUE TITLE
[build] ANGLE/OpenGL ES 2.0 using `build.zig`

### ISSUE THREAD

**Issue by wwderw** [raysan5/raylib#4248]
When using zig build to create a windows version to link to Angle, it shows that it's looking for gles2.h under the systems glad/gles.2 headers.  I looked at rlgl.h and it appears to have it set to look at gles.h in raylib's externals folders.  Is there another flag that I'm not doing to build for Window's angle support?  It does seem to work under the Linux build so far that I can tell.  Regular builds are fine for both.

Thoughts?

**Comment by raysan5** [comment]
@wwderw I'm afraid it has not been done before, someone from Zig community should probably review it...

**Comment by wwderw** [comment]
I have to wonder if it could be a combination of "skill issues" and/or the ever moving target of stable zig.  I updated to one of the dailies today and even parts of the script for my linux build got broken.  So I decided to try invoking zig cc directly and I was able to cross compile without a problem.  I may just try to write a nims script so I don't always have to go through that for every project, but at least it's something.

May just have to wait until zig is stable to try to working with build.zig.  Apparently I'm not smart enough to get around that one.  Which I never did quite learn Make and Cmake, had to use Premake.  That I could handle.

Thanks for the help.
